source/_posts/2017-02-07-falsehoods-programmers-believe-about-language.markdown

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "Falsehoods Programmers Believe About Language"
+date: 2017-02-07 22:17:11 -0600
+comments: true
+categories: 
+---
+
+As is now some kind of traditoin akin to "X Considered Harmful" posts of old,
+here I am, a software developer, writing a post trying to lay bare some of the
+[wrong ideas many developers hold about a broad topic](https://github.com/jameslk/awesome-falsehoods).
+In keeping with my degree and a personal area of interest, let's look at some of
+the things (Anglophonic) developers often believe about language that do not
+_quite_ track to reality.
+
+
+## Translating my application into other languages isn't important.
+
+It's true that many people in the world who speak more than one language choose
+to learn English and that many of those people speak "better" English than many
+native English speakers. (I put "better" in quotes because that idea is pretty
+judgemental against certain dialects of English that often have racist and/or
+classist components).
+
+However, there's a big difference between being _able_ to use an application and
+being _comfortable_ using an application. If two competing applications exist
+and one is in a user's native language, they are very likely to pick that one
+even if it doesn't have all the bells and whistles.
```

Comment:
I don't think this point is wrong, but citations/numbers would help make this point stronger. Maybe an example of a worse product winning in a geography because it was internationalized?